### PR TITLE
ACP-16: Show error on failure

### PR DESCRIPTION
**Pull request by jonholmberg**

### CODE REVIEW

File: Customers/app/src/main/java/com/e_conomic/customers/ui/fragments/SimpleAlertFragment.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * Created by jonholmberg on 18/05/2017.
+ */
+class SimpleAlertFragment(val messageId: Int? = null)  : DialogFragment() {
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        val dialogBuilder = AlertDialog.Builder(activity)
+        dialogBuilder.setTitle(R.string.something_went_wrong)
+                .setPositiveButton(R.string.ok, {
+                    _, _ ->
+                    dismiss()
```
Comment (jonholmberg) [inline Customers/app/src/main/java/com/e_conomic/customers/ui/fragments/SimpleAlertFragment.kt]: Is this lambda too confusing to read?

Comment (tudorgk) [inline Customers/app/src/main/java/com/e_conomic/customers/ui/fragments/SimpleAlertFragment.kt]: It's fine. You can put the `_ ,_ ->` next to the `{` up top.

File: Customers/app/src/main/java/com/e_conomic/customers/utils/typeAliases.kt [hunk 2]
```diff
@@ -5,4 +5,4 @@ package com.e_conomic.customers.utils
  */
 
 typealias SuccessBlock = (String?) -> Unit
-typealias FailureBlock = () -> Unit
\ No newline at end of file
+typealias FailureBlock = (ERROR) -> Unit
```
Comment (tudorgk) [inline Customers/app/src/main/java/com/e_conomic/customers/utils/typeAliases.kt]: `ERROR` should be an `Exception` subclass and with normal sentence casing. Usually all caps are reserved for enums.

Comment (jonholmberg) [inline Customers/app/src/main/java/com/e_conomic/customers/utils/typeAliases.kt]: Ah but it is an enum right now. But I'll make it subclass `Exception` - sounds good :)

**Comment by jonholmberg** [comment]
How's this @tudorgk? :)

**Comment by tudorgk** [comment]
Approved 👍

**Pull request merged**